apps/investigations/kinds/manifest.cue

```diff
@@ -3,8 +3,16 @@ package investigations
 manifest: {
 	appName:       "investigations"
 	groupOverride: "investigations.grafana.app"
-	kinds: [
-		investigation,
-		investigationIndex,
-	]
-}
+	versions: {
+		"v0alpha1": {
```

Comment:
the key difference is that versions is now top-level rather than nested under each kind